src/utils/filters.ts

```diff
@@ -0,0 +1,154 @@
+// @ts-ignore
+import { geojsonToWKT } from "@terraformer/wkt";
+import { Geometry } from "geojson";
+
+export {
+  and,
+  or,
+  intersects,
+  not,
+  equals,
+  lessThan,
+  lessThanOrEqual,
+  greaterThan,
+  greaterThanOrEqual,
+  like,
+  between,
+};
+
+function isNumber(value: string | number) {
+  return typeof value == "number";
+}
+
+/**
+ * Joins two CQL filters with an AND
+ *
+ * @param {string} filter1 - A CQL filter
+ * @param {string} filter2 - A CQL filter
+ * @return {string} - Combined CQL filter string
+ */
+function and(filter1: string, filter2: string): string {
+  return filter1 + "%20AND%20" + filter2;
```

Comment:
Super small, but could we simply use a space (instead of %20) and rely on the requests module to encode for us? Might make it a tad easier to read

Reply:
It's a good point - I'll have a play around with this if we decide to carry on with the filter abstraction!